ansible_rulebook/engine.py

```diff
@@ -328,10 +330,11 @@ async def call_action(
         hosts: List,
         facts: Dict,
         rules_engine_result,
-    ) -> Dict:
+    ) -> None:
 
         logger.info("call_action %s", action)
 
+        result = None
```

Comment:
@bzwei Should the action in builtin_actions be called before we create a task, so we don't have to just create a task to say that the action is not supported or not defined.